openassessment/workflow/serializers.py

```diff
@@ -14,6 +14,10 @@ class AssessmentWorkflowSerializer(serializers.ModelSerializer):
     Serialize a AssessmentWorkflow' model.
     """
     score = serializers.ReadOnlyField(required=False)
+    assessment_score_priority = serializers.SerializerMethodField()
```

Comment:
Another way to do this is:
`assessment_score_priority = serializers.ReadOnlyField(source='ASSESSMENT_SCORE_PRIORITY')`